sdk/eventhub/Azure.ResourceManager.EventHubs/tests/Tests/ConsumerGroupTests.cs

```diff
@@ -36,22 +36,6 @@ public async Task CreateNamespaceAndGetEventhubCollection()
             _consumerGroupCollection = _eventHub.GetEventHubsConsumerGroups();
         }
 
-        [TearDown]
```

Comment:
Removed all this kind of tear down cleanup code as we already have the resource group cleanup setting.